from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_C(object):
    def setupUi(self, C):
        C.setObjectName("C")
        C.setFixedSize(791, 461)
        C.setStyleSheet("background: rgb(255, 255, 255)")
        self.centralwidget = QtWidgets.QWidget(C)
        self.centralwidget.setObjectName("centralwidget")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(150, 20, 131, 111))
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(72)
        font.setBold(True)
        font.setWeight(75)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(250, 40, 321, 81))
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(72)
        font.setBold(True)
        font.setItalic(False)
        font.setWeight(75)
        self.label_2.setFont(font)
        self.label_2.setObjectName("label_2")
        self.label_3 = QtWidgets.QLabel(self.centralwidget)
        self.label_3.setGeometry(QtCore.QRect(50, 20, 91, 111))
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(72)
        font.setBold(True)
        font.setWeight(75)
        self.label_3.setFont(font)
        self.label_3.setObjectName("label_3")
        self.label_4 = QtWidgets.QLabel(self.centralwidget)
        self.label_4.setGeometry(QtCore.QRect(50, 130, 181, 281))
        self.label_4.setObjectName("label_4")
        self.label_5 = QtWidgets.QLabel(self.centralwidget)
        self.label_5.setGeometry(QtCore.QRect(310, 120, 411, 271))
        self.label_5.setObjectName("label_5")
        C.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(C)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 799, 25))
        self.menubar.setObjectName("menubar")
        C.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(C)
        self.statusbar.setObjectName("statusbar")
        C.setStatusBar(self.statusbar)

        self.retranslateUi(C)
        QtCore.QMetaObject.connectSlotsByName(C)

    def retranslateUi(self, C):
        _translate = QtCore.QCoreApplication.translate
        C.setWindowTitle(_translate("C", "탄소"))
        self.label.setText(_translate("C", "C"))
        self.label_2.setText(_translate("C", "탄소"))
        self.label_3.setText(_translate("C", "6."))
        self.label_4.setText(_translate("C", "<html><head/><body><p><span style=\" font-size:14pt; font-weight:600;\">원자 번호  : 6번</span></p><p><span style=\" font-size:14pt; font-weight:600;\">원자 기호  : C</span></p><p><span style=\" font-size:14pt; font-weight:600;\">영어 이름  : Carbon</span></p><p><span style=\" font-size:14pt; font-weight:600;\">한글 이름  : 탄소</span></p><p><span style=\" font-size:14pt; font-weight:600;\">전자  수   : 6개</span></p><p><span style=\" font-size:14pt; font-weight:600;\">양성자 수  : 6개</span></p></body></html>"))
        self.label_5.setText(_translate("C", "<html><head/><body><p><span style=\" font-size:14pt; font-weight:600;\">원자량     : 12.0107(g/mol)</span></p><p><span style=\" font-size:14pt; font-weight:600;\">밀도       : 2.26(g, Cm^3)</span></p><p><span style=\" font-size:14pt; font-weight:600;\">녹는점     : 3550.0˚C = 6422.0˚F = 3823.15K</span></p><p><span style=\" font-size:14pt; font-weight:600;\">끓는점     : 4027.0˚C = 7280.6˚F = 4300.15K</span></p><p><span style=\" font-size:14pt; font-weight:600;\">물질의 상태: 고체</span></p></body></html>"))



if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    C = QtWidgets.QMainWindow()
    ui = Ui_C()
    ui.setupUi(C)
    C.show()
    sys.exit(app.exec_())
